docs/source/modules/tcms.core.management.commands.rst

```diff
@@ -16,3 +16,4 @@ Submodules
    tcms.core.management.commands.migrations_order
    tcms.core.management.commands.refresh_permissions
    tcms.core.management.commands.set_domain
+   tcms.core.management.commands.upgrade
```

Comment:
Also need to refresh the docs: https://kiwitcms.readthedocs.io/en/latest/installing_docker.html#upgrading